### ISSUE TITLE
Provisionierungs-URL ungeschützt

### ISSUE THREAD

**Issue by krissi** [amooma/GS4#1]
Es ist möglich die Provisionierungs-URL von einem "unberechtigten" Gerät (z.b. mit einem Webbrowser) aufzurufen: https://192.168.1.XX/settings-00041326XXXX. Die Konfiguration wird genauso ausgeliefert, wie sie das Telefon mit der angegeben Mac-Adresse bekommt, inkl. allen Passworten.

Man könnte man ein Provisionierungs-Passwort pro Telefon einbauen: https://192.168.1.XX/settings-00041326XXXX-3255e5acced6e40fc7c73ac6eaa34cdc welche man im Telefon individuell einträgt. Da schiesst man sich allerdings ggf. bei DHCP-Provisionierung ins Bein..

**Comment by wintermeyer** [comment]
Ja, das ist ein Problem. Ist jetzt die Frage, ob wir die 4er Version bis zum Lösen des Problemes vom Server nehmen sollen.
